Extensions/xkit_preferences.js

```diff
@@ -61,6 +61,9 @@ XKit.extensions.xkit_preferences = new Object({
 
 		$("#account_button").before(m_html);
 		$("#account_button > button").attr("tabindex", "8");
+		$("header div div:has([aria-label='Account'])").before(m_html);
```

Comment:
i think this is out of scope for this PR, but it's important to acknowledge that if we're going to rely on aria labels going forward we'll need to create a dictionary for them, since they differ across dashboard languages

xkit only being in english is bad enough, xkit only working if you use english tumblr would be awful
(right now this works on english, german and dutch tumblr)